src/main/kotlin/no/nav/fo/forenkletdeploy/service/TeamCustomizer.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.fo.forenkletdeploy.service
+
+import no.nav.fo.forenkletdeploy.ApplicationConfig
+import no.nav.fo.forenkletdeploy.Commit
+
+interface TeamCustomizer {
+    fun filterCommits(commits: List<Commit>, applicationConfig: ApplicationConfig): List<Commit> = commits
+}
+
+class TeamPAMAasmundCustomizer : TeamCustomizer {
+    enum class KANDIDATSOK(val appname: String) {
+        ARBEIDSGIVER("pam-kandidatsok"),
+        VEILEDER("pam-kandidatsok-veileder")
```

Comment:
Er det ikke litt overkill med enum når koden under egentlig kun trenger to konstanter?

Reply:
Det er litt overkill, men jeg er glad i `enum` for å gruppere konstanter sammen. Det blir også én mindre import i testklassen som også bruker enumet.

Reply:
👍